main.go

```diff
@@ -110,11 +120,43 @@ func main() {
 	}
 
 	if !useTLS {
-		log.Info("serving gRPC-Gateway and OpenAPI Documentation on http://", gatewayAddr)
-		log.Fatalln(gwServer.ListenAndServe())
+		logger.Info("serving gRPC-Gateway and OpenAPI Documentation", zap.String("gatewayAddr", "http://"+gatewayAddr))
+		err = gwServer.ListenAndServe()
+		if err != nil {
+			logger.Fatal("failed to serve gRPC-Gateway", zap.Error(err), zap.Bool("tls", false))
```

Comment:
Maybe I'm weird, but I do not like the fact that we need to wrap error to some `zap.Error()`
As I can see you can create use some `SugaredLogger` from zap logger which is more similar to other logger I saw. It has funcs like `Errorf`, `Infof`, etc...

But maybe I have some kind of OCD, so we can ask other peoples' thought :smile:

Reply:
It's not worth to to use sugared logger for it.
We use it only in few places, which is not so hard and annoying, but it give good performance